subsys/bluetooth/host/conn_internal.h

```diff
@@ -2,6 +2,8 @@
  *  @brief Internal APIs for Bluetooth connection handling.
  */
 
+#include <bluetooth/iso.h>
+
 /*
  * Copyright (c) 2015 Intel Corporation
  *
```

Comment:
Fix copyright, add Nordic copyright line, and have the lines at the top of the file.